src/main/java/TennisGame2.java

```diff
@@ -1,129 +1,124 @@
-
+import java.util.*;
 public class TennisGame2 implements TennisGame
 {
-    public int P1point = 0;
-    public int P2point = 0;
-    
-    public String P1res = "";
-    public String P2res = "";
-    private String player1Name;
-    private String player2Name;
+    private class Player{
```

Comment:
cool that this is an own class now. However, it should be at the end of the class